### PR TITLE
Remove youth risk assessment feature flag

### PR DESCRIPTION
**Pull request by teneightfive**
### What changed

This tidies up the code around using a feature flag for the youth risk assessment work and updates the end-to-end tests accordingly.

### Why did it change

The feature flag for rolling out the youth risk assessment is no longer needed so that code has been tidied up.

#### Automated testing

- [x] Added unit tests to cover changes
- [x] Added end-to-end tests to cover any journey changes

#### Manual testing

Has been tested in the following browsers:

- [x] Chrome
- [x] Firefox
- [ ] Edge
- [ ] Internet Explorer

### Environment variables

- [x] No environment variables were added or changed

- [ ] Documented in the [README](ministryofjustice/hmpps-book-secure-move-frontend/blob/master/README.md)
- [ ] Added to [continous integration](https://app.circleci.com/pipelines/github/ministryofjustice/hmpps-book-secure-move-frontend)
- [ ] Added to staging environment (deployment repository)
- [ ] Added to preproduction environment (deployment repository)
- [ ] Added to production environment (deployment repository)

### Other considerations

Commit messages with a `fix` or `feat` type are automatically used to generate the [changelog](ministryofjustice/hmpps-book-secure-move-frontend/blob/master/CHANGELOG.md) and
[GitHub release notes](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/releases) during the release task. Please make sure they will read well on their own in a
summary of changes and that the commit body gives a more detailed description of those changes if necessary.

- [x] No new [Personally Identifiable Information (PII)](https://support.google.com/analytics/answer/6366371?hl=en) is being sent via analytics
- [x] Update [README](ministryofjustice/hmpps-book-secure-move-frontend/blob/master/README.md) with any new instructions or tasks

### CODE REVIEW

File: test/e2e/date-select.test.js [hunk 1]
```diff
@@ -5,45 +5,55 @@ import parsers from '../../common/parsers'
...
-      selector: page.nodes.dateSelectInput,
-      value,
-    },
-  ])
-  await page.submitForm()
-  await t.expect(page.getCurrentUrl()).contains(expectedDate)
-}
-
-fixture('Jump to date').beforeEach(async t => {
-  await t.useRole(pmuUser).navigateTo(home)
```
Comment (merlinc) [inline test/e2e/date-select.test.js:32]: There's no explicit testing of jump-to-date on the pmu screen, so this navigation should be removed :+1:

File: docs/environment-variables.md [hunk 2]
```diff
@@ -92,8 +92,6 @@
 
 | Name                                    | Description                                                                  | Default |
 | :-------------------------------------- | :--------------------------------------------------------------------------- | :------ |
-| FEATURE_FLAG_YOUTH_RISK_ASSESSMENT      | Set to `true` to enabled youth risk assessment for Secure Training Centres and Secure Childrens Homes |         |
```
Comment (merlinc) [inline docs/environment-variables.md:95]: Reminder: Once this is deployed to master, we'll need a chore to go through and cleanup the envvars on all the environments

File: test/e2e/move.update.allowed.test.js [hunk 3]
```diff
@@ -1,29 +0,0 @@
-import {
```
Comment (merlinc) [inline test/e2e/move.update.allowed.test.js:1]: This file has been removed  - is it no longer needed because of the youth risk assessment feature?

**Review by merlinc — APPROVED** [review]

File: test/e2e/move.update.allowed.test.js (hunk 3 above)
Comment (teneightfive) [reply test/e2e/move.update.allowed.test.js:1]: Yes, the changes mean this isn't a relevant test any more so was cleaning it up.

**Pull request merged**